src/Commands/About/AboutVersion.php

```diff
@@ -34,7 +33,11 @@ final class AboutVersion extends Command
 {
     public const GITHUB_RELEASES_YAML_URL = 'https://api.github.com/repos/friends-of-presta/fop_console/releases/latest';
 
-    /** @var \PrestaShop\PrestaShop\Core\Addon\Module\ModuleRepository */
+    /**
+     * @var mixed
```

Comment:
I think you can use [union type](https://phpstan.org/writing-php-code/phpdoc-types#union-types) instead of `mixed`.

Reply:
I don't remember well but it seems to me I tried and phpstan didn't like that  \PrestaShop\PrestaShop\Core\Addon\Module\ModuleRepositoryInterface or \PrestaShop\PrestaShop\Core\Module\ModuleRepositoryInterface don't exist.

Reply:
Ah you're right, it's impossible there.

Reply:
I looks dirty, so I also tried but failed.
I think that the worst valid notation, gg @MeKeyCool .